src/V3LinkDot.cpp

```diff
@@ -1214,6 +1214,28 @@ class LinkDotFindVisitor final : public VNVisitor {
                         }
                     }
                 }
+                AstIfaceRefDType* const ifacerefp
+                    = LinkDotState::ifaceRefFromArray(nodep->subDTypep());
+                if (ifacerefp && !nodep->isIfaceRef()) {
+                    // create a fake cell for virtual interfaces
```

Comment:
```suggestion
                    // Create a fake cell for virtual interfaces
```

Should there be one cell for each interface var, or only one for each type of interface var?

Test case of same interface used multiple times in a module.